Sources/XCTest/XCTestCase.swift

```diff
@@ -189,7 +189,7 @@ extension XCTestCase {
     ///   between these environments. To ensure compatibility of tests between
     ///   swift-corelibs-xctest and Apple XCTest, it is not recommended to pass
     ///   explicit values for `file` and `line`.
-    public func expectationWithDescription(description: String, file: StaticString = #file, line: UInt = #line) -> XCTestExpectation {
+    public func expectation(withDescription description: String, file: StaticString = #file, line: UInt = #line) -> XCTestExpectation {
```

Comment:
Agreed. However this is the method name as generated by the importer with the March 24 toolchain from `XCTest.framework`. Do you expect additional changes to the importer which would automatically remove "with"? Or is this a situation where the Objective-C header should be annotated with the `swift_name` attribute?

(Interestingly, it appears that https://github.com/apple/swift-3-api-guidelines-review doesn't include `XCTest.framework`)